keyboards/ramonimbao/mona/v1_1/config.h

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 Ramon Imbao
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xB16B
+#define PRODUCT_ID   0x404B
+#define DEVICE_VER   0x0011
+#define MANUFACTURER Ramon Imbao
+#define PRODUCT      Mona
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D1, D5, B7, F0, F1 }
+#define MATRIX_COL_PINS { D0, D3, D2, F4, F5, F6, F7, C7, C6, B6, B5, B4, D7, D6, D4 }
+#define UNUSED_PINS
+
+/* indicator LEDs */
+#define LED_CAPS_LOCK_PIN B3
+
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
```

Comment:
This is mostly a nitpick, because of ordering.

```suggestion
/* COL2ROW, ROW2COL */
#define DIODE_DIRECTION COL2ROW

/* indicator LEDs */
#define LED_CAPS_LOCK_PIN B3

```